invenio_cli/cli.py

```diff
@@ -8,69 +8,15 @@
 
 """Invenio module to ease the creation and management of applications."""
 
-import logging
 import os
-import subprocess
-from configparser import ConfigParser
 from pathlib import Path
 
 import click
 
-from .helpers import DockerHelper, LoggingConfig, LogPipe, bootstrap, \
-    build_assets
-from .helpers import server as scripts_server
-from .helpers import setup as scripts_setup
-from .helpers import update_statics
 from .helpers.cli_config import CLIConfig
-from .helpers.commands import Commands, LocalCommands
+from .helpers.commands import Commands, ContainerizedCommands, LocalCommands
```

Comment:
```suggestion
from .helpers.commands import ContainerizedCommands, LocalCommands
```

Unused import :)

Reply:
Thought so too, but it is still used for the `demo` command: https://github.com/inveniosoftware/invenio-cli/pull/106/files/0dc28a2ee9a3e4aec4a8f5d73b7c3e40214239a0#diff-95a0b99124ee2a80a5a1ae56da7acbf9R109